<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Plot</class>
 <widget class="QMainWindow" name="Plot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>694</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>694</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>694</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>694</height>
    </size>
   </property>
   <widget class="QCustomPlot" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>776</width>
      <height>501</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_close">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>630</y>
      <width>621</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#pushButton_close {
  background-color: rgb(215, 215, 215);
  color: black;
  border: 2px solid rgb(180, 180, 180);
  border-radius: 25px;
  border-color: rgb(80, 80, 80);
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_setPlot">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>530</y>
      <width>141</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(255, 151, 57);
  color: white;
  border: 2px solid rgb(180, 180, 180);
  border-radius: 50px;
  border-color: rgb(180, 180, 180);
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
    </property>
    <property name="text">
     <string>Graph</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>530</y>
      <width>601</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <widget class="QSplitter" name="splitter_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>X max</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_xMax">
        <property name="styleSheet">
         <string notr="true">QLineEdit {
	color: black;
	background-color: rgb(230, 230, 230);
	border-radius: 10px;                     /* &lt;----  20px  */ 
	border: 3px solid rgb(180, 180, 180);
}</string>
        </property>
        <property name="text">
         <string>5</string>
        </property>
       </widget>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QSplitter" name="splitter_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Y min</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_yMin">
        <property name="styleSheet">
         <string notr="true">QLineEdit {
	color: black;
	background-color: rgb(230, 230, 230);
	border-radius: 10px;                     /* &lt;----  20px  */ 
	border: 3px solid rgb(180, 180, 180);
}</string>
        </property>
        <property name="text">
         <string>-5</string>
        </property>
       </widget>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>X min</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_xMin">
        <property name="styleSheet">
         <string notr="true">QLineEdit {
	color: black;
	background-color: rgb(230, 230, 230);
	border-radius: 10px;                     /* &lt;----  20px  */ 
	border: 3px solid rgb(180, 180, 180);
}</string>
        </property>
        <property name="text">
         <string>-5</string>
        </property>
       </widget>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSplitter" name="splitter_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Y max</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_yMax">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
	color: black;
	background-color: rgb(230, 230, 230);
	border-radius: 10px;                     /* &lt;----  20px  */ 
	border: 3px solid rgb(180, 180, 180);
}</string>
        </property>
        <property name="text">
         <string>5</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
